frontend/app/model/RichEvent.scala

```diff
@@ -14,11 +15,12 @@ object RichEvent {
     eventListUrl: String,
     termsUrl: String,
     largeImg: Boolean,
-    highlightsUrlOpt: Option[String],
+    highlightsUrlOpt: Option[HighlightsMetadata] = None,
```

Comment:
Feels a bit weird that this is still called `highlightsUrlOpt` rather than just `highlightsOpt`